charts/nexus-iq/templates/deployment.yaml

```diff
@@ -49,10 +49,16 @@ spec:
             httpGet:
               path: /ping
               port: admin
+    {{- if .Values.iq.livenessProbe }}
+            {{- toYaml .Values.iq.livenessProbe | nindent 12 }}
+    {{- end }}
```

Comment:
nit: i'd advocate to indent the templating bits to be in-line with their surroundings (further right).

Reply:
Ok, makes sense, done 😄